runtime/src/main/java/org/corfudb/runtime/collections/StreamingMetrics.java

```diff
@@ -0,0 +1,34 @@
+package org.corfudb.runtime.collections;
+
+import com.codahale.metrics.MetricRegistry;
+import com.google.gson.JsonObject;
+
+/**
+ * List of metrics captured for a stream listener.
+ * <p>
+ * Created by WenbinZhu on 11/23/20.
+ */
+public class StreamingMetrics {
+
+    private final String listenerId;
+
+    private final TimingHistogram deliveryDuration;
+
+    StreamingMetrics(StreamListener listener, String namespace, String streamTag, MetricRegistry registry) {
+        this.listenerId = String.format("StreamListener_%s_%s_%s", listener, namespace, streamTag);
```

Comment:
This won't produce any metrics. We moved away from using dropwizard to micrometer, make sure the correct registry is passed.

Reply:
Done, switched to micrometer.